config/configauth/context.go

```diff
@@ -17,21 +17,21 @@ package configauth
 import "context"
 
 var (
-	subjectKey = subjectType{}
-	groupsKey  = groupsType{}
+	SubjectKey = subjectType{}
```

Comment:
I wonder why this has to be public if we provide the assessors from the context in the methods below.

Reply:
This is currently public as the authenticators will set those keys into the context.

Reply:
Can't we instead of making them public create the method for setting the values? My main concern is that making the public makes it possible for any piece (not only authenticators) to access to access and modify this data.